src/service/poc_mobile.proto

```diff
@@ -80,6 +83,43 @@ message subscriber_location_ingest_report_v1 {
   subscriber_location_req_v1 report = 2;
 }
 
+enum signal_level {
+  NO = 0;
+  LOW = 1;
+  MEDIUM = 2;
+  HIGH = 3;
+}
+
+// Radio signal level and power in the h3 hex.
+message radio_hex_signal_level {
+  // H3 hex of the location of the radio
```

Comment:
About comment. 

It may be a hex location covered by radio, but not a location of the radio itself. 
Could you confirm? @thehardbits

Reply:
ah, right, that's an important distinction. Thanks guys!